packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -68,6 +69,83 @@ export interface DeploymentCircuitBreaker {
   readonly rollback?: boolean;
 }
 
+/**
+ * The ecs metric name
+ * Units of this metrics are in form of percentages
```

Comment:
```suggestion
 * Units of this metrics are percentages
```

Reply:
Got it!I 
I made the sentence correction.